repairnator/repairnator-core/src/main/java/fr/inria/spirals/repairnator/BuildListener.java

```diff
@@ -0,0 +1,6 @@
+package fr.inria.spirals.repairnator;
+
+/**
+ * This class fetch build ids from ActiveMQ queue and run the pipeline with it.
+ */
+public interface BuildListener {}
```

Comment:
Your interface should not be empty. According to your `ScannerBuildListener`, method `setLauncher` and `runAsConsumerServer` should be put here.